sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_integers.py

```diff
@@ -19,151 +19,147 @@
 
-    def __add__(self, other):
+    def __add__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value + (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __sub__(self, other):
+    def __sub__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value - (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __mul__(self, other):
+    def __mul__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value * (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __xor__(self, other):
+    def __xor__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value ^ (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __lshift__(self, other):
+    def __lshift__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value << (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __rshift__(self, other):
+    def __rshift__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value >> (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __and__(self, other):
+    def __and__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         result = self.value & (other.value if isinstance(other, UInt64) else other)
         return UInt64(result & 0xFFFFFFFFFFFFFFFF)
 
-    def __or__(self, other):
+    def __or__(self, other: Union[int, 'UInt64']) -> 'UInt64':
         if isinstance(other, UInt64):
             return UInt64(self.value | other.value)
         if isinstance(other, int):
             return UInt64(self.value | other)
         raise TypeError("Unsupported type for OR operation")
 
-    def __invert__(self):
+    def __invert__(self) -> 'UInt64':
         return UInt64(~self.value & 0xFFFFFFFFFFFFFFFF)
 
     @staticmethod
-    def encode_double_as_uint64(value):
+    def encode_double_as_uint64(value: Union[int, float]) -> int:
         value_in_uint64 = struct.unpack('<Q', struct.pack('<d', value))[0]
         mask = 0x8000000000000000
         return (value_in_uint64 ^ mask) if value_in_uint64 < mask else (~value_in_uint64) + 1
 
     @staticmethod
-    def decode_double_from_uint64(value):
+    def decode_double_from_uint64(value: int) -> int:
         mask = 0x8000000000000000
         value = ~(value - 1) if value < mask else value ^ mask
         return struct.unpack('<d', struct.pack('<Q', value))[0]
 
-    def __int__(self):
+    def __int__(self) -> int:
         return self.value
 
 
 class UInt128:
-    def __init__(self, low, high):
+    def __init__(self, low: Union[int, UInt64], high: Union[int, UInt64]):
```

Comment:
```suggestion
    def __init__(self, low: Union[int, UInt64], high: Union[int, UInt64]) -> None:
```

Reply:
will `UInt64` be a forward reference here similar to the other methods?

Reply:
Not needed as this is an unrelated class.